app/config/services.yml

```
@@ -16,6 +16,7 @@ services:
             - "@security.token_storage"
             - "%wallabag_core.cache_lifetime%"
             - "@translator"
+            - "@kernel"
```

Use directly the variable instead of injecting the whole class.

```suggestion
            - "%kernel.root_dir%"
```